src/Api/Auth/Models/Request/Accounts/UpdateProfileRequestModel.cs

```diff
@@ -8,13 +8,12 @@ public class UpdateProfileRequestModel
     [StringLength(50)]
     public string Name { get; set; }
     [StringLength(50)]
-    [Obsolete("Changes will be made via the 'password' endpoint going forward.")]
+    [Obsolete("This field is ignored. Changes are made via the 'password' endpoint.")]
     public string MasterPasswordHint { get; set; }
```

Comment:
❓ : is this a good time to remove this field if it is / has been flagged as obsolete for some time?

Reply:
yes, two years. Clients have not been using this endpoint to update hints for that long, too.

Reply:
Somehow I misread your comment.

I don't want to remove it since the clients are still sending this data. Instead, I've finally removed the usage from clients and we can then remove from the API shortly.

Reply:
Ah, ok! Sounds good!